composer.json

```
@@ -33,7 +33,7 @@
   "prefer-stable": true,
   "require": {
     "oat-sa/generis": "15.35.5",
-    "oat-sa/tao-core": "54.10.4.1",
+    "oat-sa/tao-core": "v54.10.4.2",
```

```suggestion
    "oat-sa/tao-core": "54.10.4.2",
```

@gabrielfs7 Why it has V in the vesrion?

---

Both tags are valid. The original one has actually needs to have a v as standard https://github.com/oat-sa/tao-core/releases/tag/v54.10.4.2

I honestly find it messy, so we have lots of inconsistencies as you can see, but I will remove the v here to keep it as it was at least for this version